charts/external-dns/values.yaml

```
@@ -213,8 +213,59 @@ txtSuffix:
 domainFilters: []
 
 provider:
-  # -- _ExternalDNS_ provider name; for the available providers and how to configure them see the [README](https://github.com/kubernetes-sigs/external-dns#deploying-to-a-cluster).
+  # -- (string) _ExternalDNS_ provider name. When the provider is `webhook` it will add a sidecar using the `webhook` values.
```

```suggestion
  # -- _ExternalDNS_ provider name. When the provider is `webhook` it will add a sidecar using the `webhook` values.
```
The type is only needed when `helm-docs` can't figure it out from the value.